devops/linux/docker/helper/scripts/d-stack-create-dirs.sh

```diff
@@ -31,6 +31,8 @@ pushd .
 
   mkdir -p ${DATA_POSTGRESQL_DIR}
   cp ./conf/postgres/*.* ${DATA_POSTGRESQL_DIR}/.
+  mkdir -p ${DATA_POSTGRESQL_DB_DIR}
+  chmod 777 ${DATA_POSTGRESQL_DB_DIR}
```

Comment:
We should use `777` for permission, To permissive, and presents a security as any random process will have full control of that folder

Reply:
p.s have update this as well 👍🏽